### PR TITLE
Implement maintenance methods

### PR DESCRIPTION
**Pull request by magik6k**

### CODE REVIEW

File: measure.go [hunk 1]
```diff
@@ -145,6 +172,45 @@ func (m *measure) Query(q query.Query) (query.Results, error) {
...
+			m.scrubErr.Inc()
+		}
+		return err
+	}
+	return nil
+}
+
+func (m *measure) CollectGarbage() error {
+	defer recordLatency(m.gcLatency, time.Now())
+	m.gcNum.Inc()
```
Comment (Stebalien) [inline measure.go]: Shouldn't we only modify the metrics if we actually do a GC? That is, shouldn't these go inside the condition?

**Review by Stebalien — APPROVED** [review]
One question but mostly LGTM.

File: measure.go (hunk 1 above)
Comment (magik6k) [reply measure.go]: The count metrics are described as `Total number of Datastore.[..] calls`, so my reasoning was that we want to count number of method invocations, not what they are actually doing.

**Review by whyrusleeping — APPROVED** [review]
LGTM, merge at will

File: measure.go (hunk 1 above)
Comment (Stebalien) [reply measure.go]: I see... Meh.

**Pull request merged**